openapi/paths/data-exports.yaml

```
@@ -45,6 +45,11 @@ post:
     - lang: JavaScript
       source:
         $ref: ../code_samples/JavaScript/data-exports/post.js
+    - lang: PHP
+      label: PHP-SDK-3.0
+      source:
+        $ref: >-
+          ../code_samples/PHP-SDK-3.0/data-exports/post.php
```

```suggestion
      source:
        $ref: ../code_samples/PHP-SDK-3.0/data-exports/post.php
```
https://github.com/Rebilly/api-definitions/blob/82fc2268e608a91bb18293a9c92442e76d92a991/openapi/paths/invoices%40%7Bid%7D%40issue.yaml#L48-L49 ?

---

Seems formatting is different compared to previous merged PHP-SDK-3.0 samples